addons/rating/controllers/main.py

```diff
@@ -25,6 +25,14 @@ def action_open_rating(self, token, rate, **kwargs):
             raise ValueError(_("Incorrect rating: should be 1, 3 or 5 (received %d)"), rate)
```

Comment:
Commit message:

> allow only assigne partner to give the rating

Typo: "assigned".

Also, maybe don't specify the exact message displayed in the template, that way we can modify it without having to modify the commit message every time.

And finally, when specifying the task ID, we prefer "Task-XXXX". If you space out the number, github may try to match it with a commit hash, which we don't want.